### PR TITLE
Cost 2118 forecast cost type

### PR DESCRIPTION
**Pull request by shannon-donahue**
Adding cost-type variables used in AWS savings plan for AWS forecasts 

Testing instructions:
1. pull branch 
2. bring up docker and create/load test costumer data
3. hit the following endpoints : The numerical values should match for unblended and blended cost, but differ for savingsplan_effective cost
```
http://127.0.0.1:8000/api/cost-management/v1/forecasts/aws/costs/?cost_type=unblended_cost
http://127.0.0.1:8000/api/cost-management/v1/forecasts/aws/costs/?cost_type=blended_cost
http://127.0.0.1:8000/api/cost-management/v1/forecasts/aws/costs/?cost_type=savingsplan_effective_cost
 
The meta data at the top should change to match the filter
"meta": {
        "count": 26,
        "cost_type": COST_TYPE
    },
```
4. for front end testing set up a new value in the settings page to test if the default link with no filters sets to the users setting
```
http://127.0.0.1:8000/api/cost-management/v1/forecasts/aws/costs/
```

#### Smoke tests results
* we reached out to QE about the failing smoke test and they have seen it in main so it is unrelated to our PR

### CODE REVIEW

**Review by dlabrecq — DISMISSED** [review]
LGTM, but unit tests failed

File: koku/api/forecast/serializers.py [hunk 1]
```diff
@@ -37,6 +38,38 @@ def validate(self, data):
 class AWSCostForecastParamSerializer(ForecastParamSerializer):
     """AWS Cost Forecast Serializer."""
 
+    COST_TYPE_CHOICE = (
```
Comment (maskarb) [inline koku/api/forecast/serializers.py:41]: Just an observation, but this tuple is now defined in 2 different places. We ought to consider consolidating these to one place and importing where it is needed in a future PR.

**Review by maskarb — APPROVED** [review]
lgtm! Nicely done.

**Pull request merged**